internal/functions/New-DbaLogShippingSecondaryDatabase.ps1

```diff
@@ -246,7 +246,7 @@ function New-DbaLogShippingSecondaryDatabase {
 
             # For versions prior to SQL Server 2014, adding a monitor works in a different way.
             # The next section makes sure the settings are being synchronized with earlier versions
-            if ($MonitorServer -and ($ServerSecondary.Version.Major -lt 12)) {
+            if ($MonitorServer -and ($SqlInstance.Version.Major -lt 12)) {
```

Comment:
Are you sure on this change? $SqlInstance is probably not an SMO, but $ServerSecondary is.

Reply:
Can confirm this is very likely not an SMO server, it's passing a `dbainstanceparameter` from `Invoke-DbaDbLogShipping`. So it'll be $ServerSecondary or $ServerPrimary

Reply:
I'm not sure what you mean with that? The command `Invoke-DbaDbLogShipping` calls the command with the `SqlInstance` parameter. I did not change the `Invoke-DbaDbLogShipping` command in that aspect so that was not working correctly as well

Reply:
No no, `Invoke-DbaDbLogShipping` is working. I was just back tracing to ensure you didn't pass in a connected object thus making `SqlInstance` valid. As it stands now, $SqlInstance will not have a Version.Major because it's a class and not a connected server.

Reply:
I updated it back to $ServerSecondary. If this needs to be changed to primary, we can do it in another PR 😇  But for sure it is not Sqlinstance.